### DISCUSSION TITLE
skos:preLabel or schema:name for vocabularies?

### DISCUSSION THREAD

**Discussion by saumier** [culturecreates/artsdata-data-model#102] [Controlled Vocabularies]
Controlled vocabularies in Artsdata, like that for Event Types, currently uses skos:prefLabel to label the concepts. Should schema:name be used instead to make the vocabulary more reusable?

**Comment by fjjulien** [comment]
What's the intent behind your question? Are you considering using `schema:name` to make the Artsdata vocabularies more user-friendly for human readers of JSON-LD?

Until the intent is clarified, here are further questions that may help guide the discussion:

1. Is `schema:name` as fit as `skos:prefLabel` for describing multi-lingual vocabularies? If not, we might want to stick with SKOS.
2. `schema:name` is a `Subproperty of: rdfs:label`. Isn't also the case with `skos:prefLabel`?
3. Is this an "either or" question or could it be a "both and" scenario? In other words, can we continue to use SKOS to structure the information following a well-kown and efficient knowledge organization system, and represent the same information in schema when generating JSON-LD structured data?
